main_logic/tts_client.py

```
@@ -22,6 +22,12 @@
 
 logger = get_module_logger(__name__, "Main")
 
+# 关闭哨兵：core.py 通过 request_queue.put((TTS_SHUTDOWN_SENTINEL, None))
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)